addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -53,6 +53,74 @@ def res_partner_enrich_and_create_company(self, partner_id):
 
         return response
 
+    @http.route('/mail_plugin/partner/enrich_and_update_company', type='json', auth='outlook', cors='*')
+    def res_partner_enrich_and_update_company(self, partner_id):
+        """
+        Enriches an existing company using IAP
+        """
+        partner = request.env['res.partner'].browse(partner_id)
+        if not partner.is_company:
+            return {'error': _('Contact must be a company')}
```

Comment:
Seems it is not already existing in translated strings. As we target stable could you check and maybe export this string in pot file ?

Reply:
Yes I think that there is no need for a translated string here.